lib/error.php

```diff
@@ -79,3 +79,34 @@ function logBridgeError($bridgeName, $code)
 
     return $report['count'];
 }
+
+function create_sane_stacktrace(\Throwable $e): array
+{
+    $frames = array_reverse($e->getTrace());
+    $frames[] = [
+        'file' => $e->getFile(),
+        'line' => $e->getLine(),
+    ];
+    $stackTrace = [];
+    foreach ($frames as $i => $frame) {
+        $file = $frame['file'] ?? '(no file)';
+        $line = $frame['line'] ?? '(no line)';
+        $stackTrace[] = sprintf(
+            '#%s %s:%s',
```

Comment:
Yeah I considered it. But the stacktrace become so verbose.

Reply:
Well, without it, it is a lot harder to reason about, especially when you have multiple frames in the same file – symbol names are much easier to carry in head than line numbers.

Reply:
Most of the time we want the line number in the offending bridge. But I agree that class + function + function-type is nice to have too.

Reply:
Yeah, I am mostly looking at it from perspective of developer debugging an issue, it is not very useful to user reporting it – but it will likely still be useful to the developer reading the report.